en/messages.json

```diff
@@ -244,12 +248,15 @@
     "popupCriticalEmailForwardingHeadline": {
         "message": "Critical email forwarding"
     },
-    "popupBlockPromotionalEmailsHeadline_2": {
+    "popupPromoBlockingButtonTitle": {
         "message": "Block promotional emails"
     },
-    "popupBlockPromotionalEmailsBody_mask": {
+    "popupPromoBlockingButtonCopy": {
         "message": "Enable Block Promotional Emails on a mask to stop marketing emails from reaching your inbox."
     },
+    "popupPromoBlockingButtonLearnMore": {
+        "message": "Learn more"
+    },
```

Comment:
Context: This hover/tooltip when creating a custom mask: 

<img width="366" alt="image" src="https://user-images.githubusercontent.com/2692333/224164464-1e01eb66-1002-484c-b5ba-8db62abf6081.png">